Assets/Scripts/StreetBuilder.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public sealed class StreetBuilder : MonoBehaviour {
+
+	//VARIAÇÕES
+	public GameObject cityStreet;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://www.codacy.com/app/mendes-barreto/BSToysGame/pullRequest?prid=684733)